### ISSUE TITLE
Command failed: jest --config e2e/config.json --testNamePattern '^((?!:ios:).)*$' e2e

### ISSUE THREAD

**Issue by Erko21** [wix/Detox#3364]
### Description

When i try to run detox test, I always getting an error:

```
$ detox test --configuration android
12:21:03.939 detox[10976] INFO:  [test.js] DETOX_CONFIGURATION="android" DETOX_REPORT_SPECS=true DETOX_START_TIMESTAMP=1651224063897 DETOX_USE_CUSTOM_LOGGER=true jest --config e2e/config.json --testNamePattern '^((?!:ios:).)*$' e2e
.)*$' was unexpected at this time.

C:\Users\olena\Documents\Simple2b\kryptr>  "C:\Users\olena\Documents\Simple2b\kryptr\node_modules\.bin\\node.exe"  "C:\Users\olena\Documents\Simple2b\kryptr\node_modules\.bin\\..\jest-expo\bin\jest.js" --config e2e/config.json --testNamePattern '((?!:ios:).)*$' e2e
12:21:04.073 detox[10976] ERROR: [cli.js] Command failed: jest --config e2e/config.json --testNamePattern '^((?!:ios:).)*$' e2e
```

### Your environment

Detox version: 19.6.5
React Native version: 0.67.3 
Node version: 16.14.0
Device model: android.emulator :Pixel_5_API_30
OS: windows
Test-runner (select one): jest-circus

**Comment by sjransom** [comment]
Which version of `jest-circus` are you using?

I had a similar issue with the latest version so downgraded which fixed this for me:
`"jest-circus": "27.5.1"`

**Comment by d4vidi** [comment]
So indeed this looks like something that could be Windows-specific.
@noomorph could we add this to the pool?

**Comment by Erko21** [comment]
> Which version of `jest-circus` are you using?
> 
> I had a similar issue with the latest version so downgraded which fixed this for me: `"jest-circus": "27.5.1"`

I have "jest-circus": "^28.0.3"

**Comment by noomorph** [comment]
@d4vidi, I don't plan to have this `testNamePattern` anymore in Detox 20. We can do it in runtime via test environment.
